mne/io/curry/tests/test_curry.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: UTF-8 -*-
+#
+# Authors: Dirk Gütlin <dirk.guetlin@stud.sbg.ac.at>
+#
+#
+# License: BSD (3-clause)
+
+import os
+import os.path as op
+import pytest
+from numpy.testing import assert_allclose, assert_array_equal
+from mne.datasets import testing
+from mne.event import find_events
+from mne.io.constants import FIFF
+from mne.io.edf import read_raw_bdf
+from mne.io.bti import read_raw_bti
+from mne.io.curry import read_raw_curry
+from mne.io.curry.curry import _check_missing_files, _read_events_curry
+
+
+data_dir = testing.data_path(download=False)
+curry_dir = op.join(data_dir, "curry")
+
+bdf_file = op.join(data_dir, 'BDF/test_bdf_stim_channel.bdf')
+
+bti_rfDC_file = op.join(data_dir, 'BTi/erm_HFH/c,rfDC')
+
+curry7_rfDC_file = op.join(curry_dir, "c,rfDC Curry 7.dat")
+curry8_rfDC_file = op.join(curry_dir, "c,rfDC Curry 8.cdt")
+
+curry7_bdf_file = op.join(curry_dir, "test_bdf_stim_channel Curry 7.dat")
+curry7_bdf_ascii_file = op.join(curry_dir,
+                                "test_bdf_stim_channel Curry 7 ASCII.dat")
+
+curry8_bdf_file = op.join(curry_dir, "test_bdf_stim_channel Curry 8.cdt")
+curry8_bdf_ascii_file = op.join(curry_dir,
+                                "test_bdf_stim_channel Curry 8 ASCII.cdt")
+
+event_file = op.join(curry_dir, "test_bdf_stim_channel Curry 7.cef")
+
+
+@pytest.mark.filterwarnings('ignore::RuntimeWarning')
+@testing.requires_testing_data
+def test_read_raw_curry():
+    """Test reading CURRY files."""
```

Comment:
All this should be parametrized. It's the same call. But I'll do it. Either pushing in your branch, making a PR to your branch, or we can merge this test as is, and I'll do a PR on the top.

Reply:
OK , thanks!

Reply:
There you go, https://github.com/DiGyt/mne-python/pull/1 

We can discuss there what does and what doesn't make sense. There are still things that can be simplified. But there are lots of changes already. Go through them and we can discuss them there.